customer-satisfaction/model/evaluator.py

```diff
@@ -4,7 +4,7 @@
 from sklearn.metrics import mean_squared_error, r2_score
 
 
-class Evaluation:
+class Evaluator:

```

Comment:
do we go with the class based approach still or refactor this a bit to be more functional?

Reply:
I don't see a reason why we would need to move away from the class based approach here

Reply:
I personally think it makes the pipeline code nicer to look at, and if somebody wants to dive deeper into what actually happens, they totally can.

Reply:
Ok